test/upsert/rocksdb-cleanup/drop-cluster-cascade.td

```diff
@@ -0,0 +1,35 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ kafka-create-topic topic=upsert
+
+> CREATE CONNECTION conn
+  FOR KAFKA BROKER '${testdrive.kafka-addr}'
+
+> DROP CLUSTER IF EXISTS c1 CASCADE;
+
+> CREATE CLUSTER c1 REPLICAS (replica1 (SIZE '1'));
+
```

Comment:
Please create a separate source that you are not `DROP`-ing and test that its files are still around after any cleanup.